### PR TITLE
Add sqlserver input plugin

### PR DESCRIPTION
**Pull request by zensqlmonitor**

### CODE REVIEW

File: plugins/inputs/sqlserver/sqlserver.go [hunk 1]
```diff
@@ -0,0 +1,1150 @@
...
+var defaultConnectionString = &Server{ConnectionString: "Server=.;app name=telegraf;log=1;"}
+
+var sampleConfig = `
+  # specify instances 
+  # All connection parameters are optional. 
+  # By default, the host is localhost, listening on default port, TCP 1433
+  # and, for Windows, the user is the currently running AD user (SSO).
+  # See https://github.com/denisenkom/go-mssqldb for detailed connection parameters.
+  
+  [[inputs.sqlserver.servers]]
```
Comment (sparrc) [inline plugins/inputs/sqlserver/sqlserver.go]: Make this a list of server strings, like this:

```
servers = ["Server=192.168.1.10;Port=1433;User Id=telegraf;Password=T$l$gr@f69*;app name=telegraf;log=1;"]
```

**Comment by sparrc** [comment]
@zensqlmonitor can you write unit tests?

**Comment by zensqlmonitor** [comment]
@sparrc, yes I can.
Test and debug have been made, first in pure SQL, and with the flags supplied by the main binary.
I made a unit test file in the first commit here, but it failed in circle ci: no docker available for sql server. How do you see this test file ? Only short test with mocking ? Thanks.

**Comment by sparrc** [comment]
@zensqlmonitor I think it would need to be a short test with mocking. You don't have to go crazy with it, but as much as you can.

All of the long unit tests rely on docker containers, which I don't think will be available for SQL Server.

**Comment by sparrc** [comment]
@zensqlmonitor Why are you using your own fork for the dependency? github.com/zensqlmonitor/go-mssqldb

If there is a feature, could you send up a PR to the original library so we can use that?

**Comment by zensqlmonitor** [comment]
@sparrc I had to comment 2 lines in the original lib as a workaround for using T-SQL flags (as suggested in their issues). I will make a PR soon...

**Comment by sparrc** [comment]
I'd also recommend trying to make your changes again off a fresh branch, this rebase looks pretty hairy

**Comment by sparrc** [comment]
and note that influxdb import paths need to be changed to influxdata

**Comment by zensqlmonitor** [comment]
@sparrc, here it is: https://github.com/denisenkom/go-mssqldb/pull/145

**Comment by zensqlmonitor** [comment]
Fresh branch to create

**Pull request closed without merging**